com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/bundle/resources/jsweb/dmloader.js

```diff
@@ -187,7 +187,7 @@ var EngineLoader = {
             function(response) {
                 var tag = document.createElement("script");
                 tag.text = response;
-                document.head.appendChild(tag);
```

Comment:
document.head is actually read-only! Browsers seem to silently accept this anyway when running from a non-sandboxed iframe.